packages/lodestar/README.md

```diff
@@ -3,42 +3,51 @@
 [![Discord](https://img.shields.io/discord/593655374469660673.svg?label=Discord&logo=discord)](https://discord.gg/aMxzVcr)
 ![ETH2.0_Spec_Version 0.9.4](https://img.shields.io/badge/ETH2.0_Spec_Version-0.9.4-2e86c1.svg)
 
-Typescript imlementation of the Eth2.0 beacon chain
 
-## What you need
-You will need to go over the [specification](https://github.com/ethereum/lodestar-specs). You will also need to have a [basic understanding of sharding](https://github.com/ethereum/wiki/wiki/Sharding-FAQs). Note that that the specification is an ongoing document and will get outdated. The reference implementation by the Ethereum development team is written in Python and can be found [here](https://github.com/ethereum/beacon_chain).
+## Prerequisites
 
-## Run dev
+* [Lerna](https://github.com/lerna/lerna)
+* [Yarn](https://yarnpkg.com/)
 
-` ./bin/lodestar dev -v 8 -p minimal`
 
-#### Using start-multi.sh
 
-`sh ./start-multi.sh <number_of_beacon_nodes> <total_validators> <validator_distribution_per_node>`
-`sh ./start-multi.sh 4 8 8,0,0,0`
+## What you need
+You will need to go over the [specification](https://github.com/ethereum/lodestar-specs). You will also need to have a [basic understanding of sharding](https://github.com/ethereum/wiki/wiki/Sharding-FAQs). Note that that the specification is an ongoing document and will get outdated. The reference implementation by the Ethereum development team is written in Python and can be found [here](https://github.com/ethereum/beacon_chain).
```

Comment:
I think this paragraph is almost entirely stale (I see it was just moved). Might think about removing entirely, or only leaving the first sentence. The link for the spec is also https://github.com/ethereum/eth2.0-specs

Reply:
Roger that, will amend